### ISSUE TITLE
look into dateparser behavior

### ISSUE THREAD

**Issue by jamwalla** [Recidiviz/recidiviz-data#1084]
`aggregate_ingest` should probably specify `languages=['en']` to dateparser. `converter_utils.parse_days` may need to be reconsidered, since neither of the first two statements below actually do what we want:
```
>>> dateparser.parse("1y 0m 0d")
datetime.datetime(2019, 2, 25, 17, 59, 58, 626324)
>>> dateparser.parse("1y 0m 0d", languages=['en'])
>>> dateparser.parse("1year 0months 0days")
datetime.datetime(2018, 2, 25, 18, 0, 8, 540492)
```

**Comment by colincadams** [comment]
is that format used somewhere? If so we might just have to supply a manual format to dateparser for that particular case, or munge the string a bit before sending it to dateparser

**Comment by colincadams** [comment]
reassigning to @jamwalla for now to clarify whether the format from the initial comment needs to be supported

**Comment by jamwalla** [comment]
yep, `0y 0m 0d` is the format for jailtracker's "Sentence" column

**Comment by jamwalla** [comment]
fixed in #1259
